src/test/scala/org/constellation/p2p/PeerAPITest.scala

```diff
@@ -55,7 +54,7 @@ class PeerAPITest
         responseAs[FinishedCheckpointAck] shouldEqual FinishedCheckpointAck(true)
       }
 
-      //      requires mockito 1.4.x and migrating all IdiomaticMockitoFixtures
+      //      requires mockito 1.4.x and migrating all IdiomaticMockitos
```

Comment:
I upgraded us to 1.4 -- what was supposed to be done here? @marcinwadon @wkoszycki

Reply:
afaik `IdiomaticMockitoFixture` is deprecated in favor of `IdiomaticMockito`